lnwallet/size.go

```diff
@@ -0,0 +1,117 @@
+package lnwallet
+
+import (
+	"github.com/roasbeef/btcd/blockchain"
+)
+
+const (
+	WitnessFactor = blockchain.WitnessScaleFactor
+	MaxTransactionCost = blockchain.MaxBlockCost
+
+	// Below you can see the structure of a Commitment transaction which
+	// will be broadcasted in case of force closure of channel. We should
+	// estimate its size rather than cooperative closure transaction,
+	// because this transaction is bigger.
+
+	// The weight(cost), which is different from the !size! (see BIP-141),
+	// is calculated as:
+	// Weight = 4 * BaseSize + WitnessSize (weight).
+	// BaseSize - size of the transaction without witness data (bytes).
+	// WitnessSize - witness size (bytes).
+	// Weight - the metric for determining the cost of the transaction.
+
+	// PW2SH:
+	//	- OP_0: 1 byte
+	//	- WitnessScriptSHA256: 32 bytes
+	P2SHSize = 1 + 32
+
+	// P2PKH:
+	//	- OP_0: 1 byte
+	//	- PublicKeyHASH160: 20 bytes
+	P2PKHSize = 1 + 20
+
+	// MultiSig:
+	//	- OP_2: 1 byte
+	//	- pubKeyAliceLength: 1 byte
+	//	- pubKeyAlice: 33 bytes
+	//	- pubKeyBobLength: 1 byte
+	//	- pubKeyBob: 33 bytes
+	//	- OP_2: 1 byte
+	//	- OP_CHECKMULTISIG: 1 byte
+	MultiSigSize = 1 + 1 + 33 + 1 + 33 + 1 + 1
+
+
+	// Witness:
+	//	- nil byte: 1 byte
+	//	- sigAlice: 73 bytes
+	//	- sigBob: 73 bytes
+	//	- WitnessScript (MultiSig)
+	WitnessSize = 1 + 73 + 73 + MultiSigSize
+
+	// FundingInput:
+	//	- PreviousOutPoint:
+	//		- Hash: 32 bytes
+	//		- Index: 4 bytes
+	//	- ScriptSig: 0 bytes
+	//	- Witness <----	we use "Witness" instead of "ScriptSig" for
+	// 			transaction validation, but "Witness" is stored
+	// 			separately and cost for it size is smaller. So
+	// 			we separate the calculation of ordinary data
+	// 			from witness data.
+	//	- Sequence: 4 bytes
+	FundingInputSize = 32 + 4 + 4
+
+
+	// OutputPayingToThem:
+	//	- Value: 8 bytes
+	//	- PkScriptLength: 8 bytes
+	//	- PkScript (PW2SH)
+	OutputPayingThemUsSize = 8 + 8 + P2SHSize
+
+	// OutputPayingUs:
+	//	- Value: 8 bytes
+	//	- PkScriptLength: 8 bytes
+	//	- PkScript (P2PKH)
+	OutputPayingToUsSize = 8 + 8 + P2PKHSize
+
+	// HTLCOutput:
+	//	- Value: 8 bytes
+	//	- PkScriptLength: 8 bytes
+	//	- PkScript (PW2SH)
+	HTLCSize = 8 + 8 + P2SHSize
+
+	// CommitmentTransaction:
+	//	- Version: 4 bytes
+	//	- Flag: 1 byte
+	//	- Marker: 1 byte
+	//	- TxIn:
+	//		FundingInput
+	//	- TxOut:
+	//		OutputPayingToThem,
+	//		OutputPayingToUs,
+	//		....HTLCOutputs...
+	//	- LockTime: 4 bytes
+	BaseCommitmentTxSize = 4 + 1 + 1 + FundingInputSize +
+		OutputPayingThemUsSize + OutputPayingToUsSize
+)
+
+
+// EstimateCommitmentTxCost returns a worst case serialize case estimate for a
+// commitment transaction.
+func EstimateCommitmentTxCost(lc *LightningChannel) (bs, ws int) {
```

Comment:
Name returned aren't used any where else within the project currently. I find that their usage can make properly following the execution of code more difficult. 

Can you remove the usage of named returns in this function? Thanks!